# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLLibraryEGL.h
# Commit: bbfe23c61add
# Full Hash: bbfe23c61add07377daf80696fd6996c0463f5b6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 8fcfac450ecc -r bbfe23c61add gfx/gl/GLLibraryEGL.h
--- a/gfx/gl/GLLibraryEGL.h	Mon Jun 08 20:13:01 2020 +0000
+++ b/gfx/gl/GLLibraryEGL.h	Mon Jun 08 20:34:15 2020 +0000
@@ -385,21 +385,21 @@
     return mIsWARP;
   }
 
-  bool HasKHRImageBase() {
+  bool HasKHRImageBase() const {
     return IsExtensionSupported(KHR_image) ||
            IsExtensionSupported(KHR_image_base);
   }
 
-  bool HasKHRImagePixmap() {
+  bool HasKHRImagePixmap() const {
     return IsExtensionSupported(KHR_image) ||
            IsExtensionSupported(KHR_image_pixmap);
   }
 
-  bool HasKHRImageTexture2D() {
+  bool HasKHRImageTexture2D() const {
     return IsExtensionSupported(KHR_gl_texture_2D_image);
   }
 
-  bool HasANGLESurfaceD3DTexture2DShareHandle() {
+  bool HasANGLESurfaceD3DTexture2DShareHandle() const {
     return IsExtensionSupported(ANGLE_surface_d3d_texture_2d_share_handle);
   }
 